.github/pull_request_template.md

```diff
@@ -1 +1,96 @@
+Please select the template that's relevant for your situation and delete the other templates.
```

Comment:
```suggestion
Please use the template that's relevant for your situation and delete the other templates.
```

Minor nit! (The GitHub issue interface lets you actually SELECT an issue template, like with a separate button for each template, so that's what the current wording made me think of.)